packages/mosaic/core/common-behaviors/color.ts

```diff
@@ -19,6 +19,7 @@ export enum ThemePalette {
     Second = 'second',
     Error = 'error',
     Default = 'second',
+    Transparent = 'transparent',
```

Comment:
it is bad idea, we dont have palette `transparent`.

I dont know how do it better, we will discuss it today.

Reply:
good point

Reply:
@lskramarov, as an option: 
- set this button with secondary color scheme (beacause it's closer by style to secondary) and add a class "transparent": `<button color="secondary" class="transparent">`

Also let's figure out - is button control - the only tranparent control for now? If yes, then this solution seems to be reasonable.

Reply:
yes, we had discussed it and decided do it with class modificator `.mc-button_transparent`, this class should work with any color.

Reply:
@lskramarov updated with new approach.